packages/ocaml-topexpect/ocaml-topexpect.0.3/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Frederic Bour <frederic.bour@lakaban.net>"
+authors: [
+  "Frederic Bour <frederic.bour@lakaban.net>"
+  "Anil Madhavapeddy <anil@recoil.org>"
+]
+homepage: "https://github.com/let-def/topexpect"
+bug-reports: "https://github.com/let-def/topexpect"
+license: "MIT"
+dev-repo: "https://github.com/let-def/topexpect.git"
+build: [make]
+depends: [
+  "ocamlfind" {build}
+  "ppx_sexp_conv" {build}
+  "ppx_deriving" {build}
```

Comment:
missing a jbuilder dep here, fixed in https://github.com/ocaml/opam-repository/pull/10138